ui/imports/shared/views/AssetsView.qml

```diff
@@ -217,16 +269,6 @@ ColumnLayout {
                     onClicked: Global.openPopup(communityInfoPopupCmp)
                 }
             }
-
-            StatusListView {
```

Comment:
I don't understand, why is this removed?

Reply:
https://github.com/status-im/status-desktop/pull/12871#discussion_r1434200352